zeebe/gateway-protocol/src/main/proto/v2/audit-logs.yaml

```diff
@@ -0,0 +1,312 @@
+## Endpoint definitions
+paths:
+  /audit-logs/search:
+    post:
+      x-eventually-consistent: true
+      tags:
+        - Audit Log
+      operationId: searchAuditLogs
+      summary: Search audit logs
+      description: Search for audit logs based on given criteria.
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AuditLogSearchQueryRequest'
+      responses:
+        "200":
+          description: The audit logs search result.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuditLogSearchQueryResult'
+        "400":
+          $ref: 'common-responses.yaml#/components/responses/InvalidData'
+        "401":
+          $ref: 'common-responses.yaml#/components/responses/Unauthorized'
+        "403":
+          $ref: 'common-responses.yaml#/components/responses/Forbidden'
+        "500":
+          description: An internal error occurred while processing the request.
+
+  /audit-logs/{operationKey}:
+    get:
+      x-eventually-consistent: true
+      tags:
+        - Audit Log
+      operationId: getAuditLog
+      summary: Get audit log
+      description: Get an audit log by operationKey.
+      parameters:
+        - name: operationKey
+          in: path
+          required: true
+          description: The audit log operation key.
+          schema:
+            $ref: 'keys.yaml#/components/schemas/AuditLogOperationKey'
+      responses:
+        "200":
+          description: The audit log is successfully returned.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuditLogResult'
+        "401":
+          $ref: 'common-responses.yaml#/components/responses/Unauthorized'
+        "403":
+          $ref: 'common-responses.yaml#/components/responses/Forbidden'
+        "404":
+          description: The audit log with the given operation key was not found.
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'problem-detail.yaml#/components/schemas/ProblemDetail'
+        "500":
+          $ref: 'common-responses.yaml#/components/responses/InternalServerError'
+
+## Schema definitions
+
+components:
+  schemas:
+    AuditLogResult:
+      description: Audit log item.
+      type: object
+      properties:
+        operationKey:
+          allOf:
+            - $ref: 'keys.yaml#/components/schemas/AuditLogOperationKey'
+          description: The unique operation key of the audit log entry.
+        entityKey:
+          type: string
```

Comment:
💭 Could this be a reference to the `LongKey`, actually? It will always point to keys that adhere to that schema, right?

Reply:
The plan is that for identity related operations this will contain the email address for instance.

[From the solution design](https://docs.google.com/document/d/1COzShIcqFqZlq6ZRfuL9VeHD7oNvVd8vhnezxGlYTU0/edit?tab=t.vpvtku4f2lqx#heading=h.kd9atrl0nn0s): `May be a Long key, or an Identity-related String ID (email)`

I'd keep it `string` for now.

Reply:
Well, we can clearly state that it only makes sense to always consider both, I guess.

Reply:
Yep, let's go with this for now. If we discover something during identity milestone, we can extend it.